applications/matter_bridge/src/ble_providers/ble_onoff_light_data_provider.cpp

```diff
@@ -195,14 +196,22 @@ int BleOnOffLightDataProvider::ParseDiscoveredData(bt_gatt_dm *discoveredData)
 	return 0;
 }
 
-void BleOnOffLightDataProvider::NotifyAttributeChange(intptr_t context)
+void BleOnOffLightDataProvider::NotifyOnOffAttributeChange(intptr_t context)
 {
 	BleOnOffLightDataProvider *provider = reinterpret_cast<BleOnOffLightDataProvider *>(context);
 
 	provider->NotifyUpdateState(Clusters::OnOff::Id, Clusters::OnOff::Attributes::OnOff::Id, &provider->mOnOff,
 				    sizeof(provider->mOnOff));
 }
 
+void BleOnOffLightDataProvider::NotifySwitchCurrentPositionAttributeChange(intptr_t context)
```

Comment:
One more minor, would you mind changing this module name from `BleOnOffLightDataProvider` to ` BleLBSDataProvider`? I think it would also follow the naming scheme for environmental profile.